README.md

```
@@ -30,6 +32,120 @@ pnpm install
+## ☁️ Deploy To Cloudflare Workers
+
+This project is configured for Cloudflare Workers deployment via Wrangler.
+
+1. Install dependencies and login to Cloudflare.
+
+```bash
+pnpm install
+pnpm wrangler login
+```
+
+2. Create a D1 database.
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)